asg/src/accesses/circuit_access.rs

```diff
@@ -80,6 +80,10 @@ impl<'a> ExpressionNode<'a> for CircuitAccess<'a> {
     }
 
     fn is_consty(&self) -> bool {
+        if let Some(target) = self.target.get() {
```

Comment:
Looks like you forgot to remove debug.

Reply:
oops that was from me testing the bug we were talking about. Thanks!